package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@viaprotocol/web3-wallets",
-  "version": "1.0.10",
```

Comment:
It would be great if you would merge these changes without changing a version and only after you will be sure that everything is fine -- merge a version update 👍🏻